akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -2,34 +2,15 @@
 
 For introduction to the Akka Cluster concepts please see @ref:[Cluster Specification](common/cluster.md).
 
-## Preparing Your Project for Clustering
+## Dependency
 
 The Akka cluster is a separate jar file. Make sure that you have the following dependency in your project:
```

Comment:
I think the wording should be exactly the same for all these

To use Akka Cluster, add the module to your project: